### ISSUE TITLE
Can't scroll down in some doc pages 

### ISSUE THREAD

**Issue by bovas85** [ohansemmanuel/react-hooks-cheatsheet#2]
On mobile and desktop mode on chrome latest on android 9 I can't scroll down the page where othe lr examples are.

Seems an issue with fixed heights or overflow being hidden. 

I can send a PR later today unless you can fix earlier

**Comment by ohansemmanuel** [comment]
@bovas85 thanks for opening the issue. I can have a look later today, however, it'd be great if you sent a PR as well 😊

**Comment by bovas85** [comment]
I'll try to once I get a moment. Crazy weekend (it was holiday on Monday here in UK)

**Comment by ohansemmanuel** [comment]
Totally get it! 😆

**Comment by aspiers** [comment]
I see the same issue.

**Comment by ohansemmanuel** [comment]
I'll look into this.

**Comment by intisavariable** [comment]
I have the same issue. If I open the page on portrait mode (snap the browser window to half a screen) it does not scroll at all. If a page is opened in landscape (fullscreen) and then resized to portrait it works fine.

**Comment by intisavariable** [comment]
I cloned the repo, will look into trying to fix it if I can figure out what's the problem, any help/suggestions welcome

**Comment by intisavariable** [comment]
Update:
The bug only occurs after opening the hamburger menu and clicking the link. 
It does not persist through the refresh.

**Comment by ohansemmanuel** [comment]
Thanks @intisavariable I've had my hands full. Any luck with this? I'll perhaps take a second look today too.

**Comment by bovas85** [comment]
the issue is clear but I have no idea how to fix it due to the abstractions on top of the docs.

the HTML tag gets a class of `with-overlay` which is not removed upon route change

**Comment by ohansemmanuel** [comment]
@bovas85 do you get the same issue with this link: https://react-hooks-cheatsheet.com/useeffect
i.e can't scroll down

**Comment by bovas85** [comment]
Yes, I can't open nested menu items as well, but the issue appears once you go to a menu and to another page. The overflow: hidden of the  html tag is not removed

**Comment by ohansemmanuel** [comment]
I'm gonna perhaps rewrite to use another documentation library. Shouldn't be as much work since the core by itself is just MDX and basic React components

**Comment by knownasilya** [comment]
Adding an `overflow-y: auto` to `#root` fixes the issue, seems like a rewriting would be more work
